lib/TWCManager/Vehicle/TeslaAPI.py

```diff
@@ -871,7 +871,7 @@ def car_api_charge(self, charge):
                         # Don't send another command to this vehicle for
                         # carApiErrorRetryMins mins.
                         vehicle.lastErrorTime = now
-                    elif apiResponseDict["response"]["result"] == False:
+                    elif not apiResponseDict["response"]["result"]:
```

Comment:
These are not the same thing - are we sure that is the intended logic? one is a comparison between a specific value, the other is an absence of "positive" value at all

Reply:
Yes I am. `not` tests for `False` and `None`. An empty dictionary returns `False` and is therefor covered by using `not` - beside that it's PEP8 compatible coding style

Reply:
Perfect, that was the exact validation I was looking for, thanks.